### ISSUE TITLE
Change the font style

### ISSUE THREAD

**Issue by wogoos** [egorgrushko/SerialMonitor#2]
Not really and issue but is it possible and if so how  can I change the font settings of the Serial monitor. Hope you can help me out 
Thanks

**Comment by DamianSuess** [comment]
Yes, Consolas instead of an italicized font would be useful at times during Arduino comms debugging
